README.Rmd

```diff
@@ -23,7 +23,7 @@ knitr::opts_chunk$set(
 
 ## Purpose
 
-The goal of `costofliving` is to provide functions and data sets to calculate the average cost of living for a single person earning a minimum wage salary across different states in the United States in 2022. This package contains data from the [MIT Living Wage Calculator](https://livingwage.mit.edu/), [the World Population Review](https://worldpopulationreview.com/), and [the Missouri Economic Research and Information Center](https://meric.mo.gov/).
+The goal of `costofliving` is to provide functions and data sets to calculate the average cost of living for a single person earning a minimum wage salary across different states in the United States in 2022. This package contains data from the [MIT Living Wage Calculator](https://livingwage.mit.edu/), [the World Population Review](https://worldpopulationreview.com/), [the Nerd Wallet](https://www.nerdwallet.com/article/taxes/federal-income-tax-brackets), and [the Labor Law Center]([https://www.laborlawcenter.com/state-minimum-wage-rates/).
```

Comment:
I think here in the README we just wanna cite the general URL (for eg https://www.nerdwallet.com/, https://www.laborlawcenter.com for the sources) for the org/website we get the info from. The specific source/URL was cited in R/process_datasets.R already. Lmk what you think about it